charts/nri-metadata-injection/tests/image_test.yaml

```diff
@@ -1,61 +0,0 @@
-suite: test docker images references
-release:
-  name: release
```

Comment:
why removing the tests?

Reply:
The test for image is done in the test for the `common-library`. That test is a copy from the `common-library` and has the version of the chart hardcoded so it has to be changed every release.

I saw it redudant.

Reply:
(even if it is true that there is not that much logic left here)

Reply:
Changing the registry is [also tested in the `common-library`](https://github.com/newrelic/helm-charts/pull/931/files#diff-10951ec324b8569dc123f3c9a008485b670caafd7f428b6c07758e5c5013a9f5) :)